packages/cxl-lumo-styles/scss/_mixins.scss

```diff
@@ -43,3 +43,65 @@
     max-width: $max-width;
   }
 }
+
+/*
+* Typography Styles mixins
+*/
+
+@mixin light {
+  font-weight: 300;
+}
+
+@mixin regular {
+  font-weight: 400;
+}
+
+@mixin bold {
+  font-weight: 900;
+}
+
+@mixin italic {
+  font-style: italic;
+}
```

Comment:
It's a wrapper for regular CSS style. Could you explain how this is better then just doing `font-style: italic;` everywhere?